### ISSUE TITLE
Unable to set the default chosen tab.

### ISSUE THREAD

**Issue by KirlosYousef** [simformsolutions/SSCustomTabbar#17]
Hello,

I have been trying to set the default chosen tab view when the app starts but it's not working, Could you please tell me how can I do that?
I am using SwiftUI

Thank you.

**Comment by akhilftech** [comment]
Just add the Below code(In my case selected tab is 2)

 override func viewDidAppear(_ animated: Bool) {
        super.viewDidAppear(animated)
            self.tabBarController?.selectedIndex = 2
        }

**Comment by vraj-s-shah** [comment]
Hi @KirlosYousef, you can use `SSTabConfiguration()` to set selected tab while using SwiftUI. Refer to the example given below:
```
let tabBarView = SwiftUITabBarController(tabItems: [vc1, vc2, vc3, vc4, vc5], configuration: .constant(SSTabConfiguration(selectedIndex = 1)), isTabBarHidden: self.$isTabBarHidden)
```
<br>
Thank you for reaching us out, hope this helps 😊.

**Comment by mobile-simformsolutions** [comment]
@KirlosYousef Functionality is available in the latest release version [2.0.7](https://github.com/SimformSolutionsPvtLtd/SSCustomTabbar/tree/2.0.7).
